src/material-experimental/mdc-menu/menu.spec.ts

```diff
@@ -1 +1,2345 @@
-// TODO: copy tests from existing mat-menu, update as necessary to fix.
+import {ComponentFixture, fakeAsync, flush, inject, TestBed, tick} from '@angular/core/testing';
+import {By} from '@angular/platform-browser';
+import {NoopAnimationsModule} from '@angular/platform-browser/animations';
+import {
+  Component,
+  ElementRef,
+  EventEmitter,
+  Input,
+  Output,
+  NgZone,
+  TemplateRef,
+  ViewChild,
+  ViewChildren,
+  QueryList,
+  Type,
+  Provider,
+} from '@angular/core';
+import {Direction, Directionality} from '@angular/cdk/bidi';
+import {OverlayContainer, Overlay} from '@angular/cdk/overlay';
+import {ESCAPE, LEFT_ARROW, RIGHT_ARROW, DOWN_ARROW, TAB, HOME, END} from '@angular/cdk/keycodes';
+import {
+  MAT_MENU_SCROLL_STRATEGY,
+  MatMenuTrigger,
+  MenuPositionX,
+  MenuPositionY,
+  MatMenuPanel,
+  MAT_MENU_DEFAULT_OPTIONS
+} from '@angular/material/menu';
```

Comment:
nothing in this test should have to import from `@angular/material/menu`

Reply:
I'm not sure how to get around it. I can make the types weaker, but I still need a reference to the injection token.

Reply:
You should re-create or re-export every symbol from `@angular/material/menu` (e.g. like I did for checkbox: https://github.com/angular/material2/blob/master/src/material-experimental/mdc-checkbox/public-api.ts)